ReplyAdaptabilityCodelab/app/src/main/java/com/example/reply/data/EmailsRepositoryImpl.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.reply.data
+
+import com.example.reply.data.local.LocalEmailsDataProvider
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+class EmailsRepositoryImpl : EmailsRepository {
+
+    override fun getAllEmails(): Flow<List<Email>> = flow {
+        delay(1500)
```

Comment:
remove the delay completely for all functions.
For what you're teaching in the codelab you don't need a delay.